tests/integration/ha_tests/helpers.py

```diff
@@ -37,6 +38,11 @@ class ProcessError(Exception):
     """Raised when a process fails."""
 
 
+def get_patroni_cluster(unit_ip: str) -> Dict[str, str]:
+    resp = requests.get(f"http://{unit_ip}:8008/cluster")
+    return resp.json()
+
+
 async def change_master_start_timeout(ops_test: OpsTest, seconds: Optional[int]) -> None:
```

Comment:
We should switch this to `primary` since we are now on patroni 3

Reply:
Removed on https://github.com/canonical/postgresql-k8s-operator/pull/134/commits/84d810d13eaedb5dc22e0cd2bc8faf6489dd0484, as the `master_start_timeout` is not being used. [DPE-1407](https://warthogs.atlassian.net/browse/DPE-1407) is in the backlog to fix the wrong wording in the other places.

Reply:
I reverted it due to changes needed to address the above comment.